### PR TITLE
Use some rich format to improve README.md readability

### PR DESCRIPTION
**Pull request by PeterDaveHello**
**Pull Request Type**

- [ ] Bugfix
- [ ] Feature Implementation
- [x] Documentation
- [ ] Other

**Description**

Use some rich format to improve the readability.

**Screenshots**

| Before | After |
| ------ | ----- |
| image | image |

### CODE REVIEW

**Review by absidue — DISMISSED** [review]

**Review by PikachuEXE — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -104,11 +104,11 @@ You can also stay up to date by reading the [FreeTube Blog](https://write.as/fre
...
-[FreeTube on Liberapay](https://liberapay.com/FreeTube)
+* [FreeTube on Liberapay](https://liberapay.com/FreeTube)
 
-Bitcoin Address: 1Lih7Ho5gnxb1CwPD4o59ss78pwo2T91eS
+* Bitcoin Address: `1Lih7Ho5gnxb1CwPD4o59ss78pwo2T91eS`
 
-Monero Address: 48WyAPdjwc6VokeXACxSZCFeKEXBiYPV6GjfvBsfg4CrUJ95LLCQSfpM9pvNKy5GE5H4hNaw99P8RZyzmaU9kb1pD7kzhCB
+* Monero Address: `48WyAPdjwc6VokeXACxSZCFeKEXBiYPV6GjfvBsfg4CrUJ95LLCQSfpM9pvNKy5GE5H4hNaw99P8RZyzmaU9kb1pD7kzhCB`
 
 If you enjoy using FreeTube, you're welcome to leave a donation using the following methods.  While your donations are much appreciated, only donate if you really want to.  Donations are used for keeping the website up and running and eventual code signing costs.
```
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [inline README.md:113]: I think its better to move this above the donate methods

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PeterDaveHello) [reply README.md:113]: Maybe in another separated change will be better?

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply README.md:113]: Could u maybe provide a visual example of that so i can decide better on that

File: README.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ addressed.
 
 FreeTube is supported by the [Privacy Redirect](https://github.com/SimonBrazell/privacy-redirect) and [LibRedirect](https://github.com/libredirect/libredirect) extension, which will allow you to open YouTube links into FreeTube. You must enable the option within the advanced settings for it to work.
```
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [inline README.md:17]: extensions instead of extension

This was missing on the previous pr. Could u address it with a commit.

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (PeterDaveHello) [reply README.md:17]: May I do that in another pull request? Kind of different type of change. ;)

Comment (absidue) [reply README.md:17]: Different pull requests and commit messages are only really necessary if you can actually tell the commits apart. As your commit messages are just "Update README.md", you might as well just stick the changes in this one.

Comment (PeterDaveHello) [reply README.md:17]: Oops, I send this patch directly on GitHub website and didn't notice that my words are in the body, should be more precisely in the subject, just updated it!

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply README.md:17]: please address this there is no need for a separate pr

File: README.md (hunk 1 above)
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply README.md:113]: @PeterDaveHello

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Review by PikachuEXE — APPROVED** [review]

**Review by absidue — APPROVED** [review]

**Pull request merged**